tests/test_convert.py

```diff
@@ -76,6 +76,69 @@ def test_readme_circuit():
     parse(source)
 
 
+def test_readme_circuit_physical_qubits():
+    source = """
```

Comment:
I'm ok to add a second integration test based on the physical-qubit mode, but let's get rid of the lines that don't make sense rather than commenting them out.

Reply:
I was wondering whether to fix them up or put in something else. But I prefer removing the cruft now.